docs/components/progress-tracker/snippets/example1.hbs

```diff
@@ -0,0 +1,26 @@
+---
+title: AXA Button
+layout: demo.hbs
+---
+
+<div class="form-steps">
+    <div class="form-steps__wrapper">
+        <div class="form-steps__prev"></div>
+        <div class="form-steps__step form-steps__step--done">
```

Comment:
Does the markup also work with inline elements? One may want to make past step's clickable (using an anchor)

Reply:
Not tested yet. This is just a first attempt but thats a good point.

Reply:
I just added the styles and an example for this use case.